# This message contains the information about the pose and the class of a single clamp.

# the pose of the clamp
geometry_msgs/Pose clamp_pose

# the class of the clamp
byte LARGE_THICK_GRAY=0
byte MEDIUM_THIN_GRAY=1
byte MEDIUM_THIN_GREEN=2
byte SMALL_THIN_BLUE=3
byte type
#string type
